packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -731,9 +731,9 @@ curl -v -X POST \
 | -------- | ------------------------------------------------------------------------------ | ---------- | ---------- | -------- |
 | type     | The authentication scheme type. Currently the only supported type is `HEADER`. | String     | TRUE       | FALSE    |
 | key      | The header name for the authorization header                                  | String     | TRUE       | FALSE    |
-| value    | The header value                                                              | String     | TRUE       | TRUE     |
+| value    | The header value. This is the secret value that you want Okta to pass to your external service endpoint (so that your external service can check for its presence as a security measure).                                                               | String     | TRUE       | TRUE     |
```

Comment:
> This secret value is passed to your external service endpoint. Your external service can check for it as a security measure.

(avoid anthropomorphizing Okta and simplified the text a bit).

Reply:
Thanks. Much better.